build/InstallerInfo.props

```diff
@@ -5,8 +5,8 @@
     <CombinedFrameworkSdkHostInstallerFile>$(InstallerOutputDirectory)/$(ArtifactNameWithVersionCombinedHostHostFxrFrameworkSdk)$(BundleExtension)</CombinedFrameworkSdkHostInstallerFile>
 
     <SdkDebianIntermediateDirectory>$(IntermediateDirectory)/debian/sdk</SdkDebianIntermediateDirectory>
-    <LayoutDirectory Condition=" '$(OSName)' == 'ubuntu' ">$(SdkDebianIntermediateDirectory)/debianLayoutDirectory</LayoutDirectory>
-    <LayoutDirectory Condition=" '$(OSName)' != 'ubuntu' ">$(IntermediateDirectory)/layouts</LayoutDirectory>
+    <LayoutDirectory Condition=" '$(OSName)' == 'ubuntu' or '$(OSName)' == 'debian' ">$(SdkDebianIntermediateDirectory)/debianLayoutDirectory</LayoutDirectory>
```

Comment:
NIT: let's be consistent on the casing for the conditionals here. Keep it all UPPER CASE. Like OR and AND.